pkgs/os-specific/darwin/linearmouse/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, stdenvNoCC, fetchurl }:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "linearmouse";
+  version = "0.9.5";
+
+  src = fetchurl {
+    url = "https://github.com/linearmouse/linearmouse/releases/download/v${finalAttrs.version}/LinearMouse.dmg";
+    hash = "sha256-T25H2CmDBt3J5zaC8gy87PyJONPsDZpF7S5eksRzgrA=";
+  };
+
+  sourceRoot = "LinearMouse.app";
+
+  # `undmg` doesn't support unpacking APFS dmgs, so we use some of the macOS built-ins to do it for us.
+  # https://discourse.nixos.org/t/help-with-error-only-hfs-file-systems-are-supported-on-ventura/25873
+  unpackCmd = ''
+    echo "File to unpack: $curSrc"
+    if ! [[ "$curSrc" =~ \.dmg$ ]]; then return 1; fi
+    mnt=$(mktemp -d -t ci-XXXXXXXXXX)
+
+    function finish {
+      echo "Detaching $mnt"
+      /usr/bin/hdiutil detach $mnt -force
```

Comment:
Hum, I  do not like that impurity...

Reply:
This can probably be replaced with _7zz. See https://github.com/NixOS/nixpkgs/pull/289900